.github/workflows/testAndPublish.yml

```
@@ -256,29 +260,25 @@ jobs:
       run: ci/scripts/installNVDA.ps1
       env:
         nvdaLauncherDir: ${{ steps.getLauncher.outputs.download-path }}
-    - name: Upload install log
-      if: ${{ success() || failure() }}
-      uses: actions/upload-artifact@v4
-      with:
-        name: Install log
-        path: |
-          testOutput/install
-          !testOutput/install/nvda_install_temp.log
     - name: Run system tests
       run: ci/scripts/tests/systemTests.ps1
       env:
         nvdaLauncherDir: ${{ steps.getLauncher.outputs.download-path }}
+        INCLUDE_SYSTEM_TEST_TAGS: ${{ matrix.testSuite }}
     - name: Upload system tests results
       if: ${{ success() || failure() }}
```

do we want to bother uploading successful test result artifacts?

---

I don't think they really give us anything
```suggestion
      if: ${{ failure() }}
```